docs/specs/manifest.yml

```diff
@@ -247,7 +247,17 @@ inputs:
     a [](b)
 outputs:
   .publish.json: |
-    { "!files": null }
+    { "files": 
```

Comment:
instead of modifying an exist spec, can you add a new spec that describes what `hasError` does

Reply:
all these modified cased are related to my changes, so I just reuse the case instead of write duplicated multiple cases